batch/src/main/resources/es_index.json

```diff
@@ -0,0 +1,349 @@
+            },
+            "entryDate": {
+              "type": "date",
+              "format": "strict_date_optional_time||epoch_millis",
+              "index": "not_analyzed"
+            },
+            "lastUpdateDate": {
+              "type": "date",
+              "format": "strict_date_optional_time||epoch_millis",
+              "index": "not_analyzed"
+            },
+            "organizationName": {
+              "type": "string",
+              "analyzer": "welsh_split_analyzer"
+            },
+            "poBoxNumber": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "postTown": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "postcode": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "postcodeType": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "proOrder": {
+              "type": "long",
+              "index": "not_analyzed"
+            },
+            "processDate": {
+              "type": "date",
+              "format": "strict_date_optional_time||epoch_millis",
+              "index": "not_analyzed"
+            },
+            "recordIdentifier": {
+              "type": "byte",
+              "index": "not_analyzed"
+            },
+            "startDate": {
+              "type": "date",
+              "format": "strict_date_optional_time||epoch_millis",
+              "index": "not_analyzed"
+            },
+            "subBuildingName": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "thoroughfare": {
+              "type": "string",
+              "index": "not_analyzed"
+            },
+            "udprn": {
+              "type": "integer",
+              "index": "not_analyzed"
+            },
+            "uprn": {
+              "type": "long",
+              "index": "not_analyzed"
+            },
+            "welshDependentLocality": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "welshDependentThoroughfare": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "welshDoubleDependentLocality": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "welshPostTown": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "welshThoroughfare": {
+              "type": "string",
+              "analyzer": "welsh_no_split_analyzer"
+            },
+            "pafAll": {
+              "type": "string",
+              "analyzer": "welsh_split_analyzer"
+            }
+          }
+        },
+        "uprn": {
+          "type": "long",
```

Comment:
Agreed. Given that the index is very small, I agree that this level of duplication does not really hurt.